WindowsServerDocs/networking/windows-time-service/Windows-Time-Service-Tools-and-Settings.md

```diff
@@ -261,6 +261,12 @@ Below you'll find the default values for the **Global Configuration Settings** o
 |Group Policy setting|Default value|
 | --- | --- |
 |AnnounceFlags|10|
+|ChainDisable|0|
+|ChainEntryTimeout|16|
+|ChainLoggingRate|30|
```

Comment:
Should we also have "ChainMaxEntries" with a default of 128:
<img width="347" alt="image" src="https://github.com/user-attachments/assets/b5911e2f-9590-4624-b3dd-1a12cc4386ce">

Reply:
Thanks!